### ISSUE TITLE
Does not work with Stable Elysia@v1

### ISSUE THREAD

**Issue by cybercoder-naj** [gaurishhs/elysia-ip#11]
IP returns `null`. There are breaking changes with Elysia@v1. See [#513](https://github.com/elysiajs/elysia/issues/513). 

### Versions

- elysia-ip: 0.0.7
- elysia: 1.0.9

### Reproduction

```ts
const app = new Elysia()
  .use(ip())
  .get('/', ({ ip }) => {
    console.log("ip", ip); // logs "ip null"
    return "Hello"
  })
```

**Comment by gaurishhs** [comment]
hi, thanks for reporting. Published a released which is working with Elysia v1. The module is also 1.0 now.

**Comment by cybercoder-naj** [comment]
![image](https://github.com/gaurishhs/elysia-ip/assets/48282810/958674ab-e8c3-44be-8cbc-e79c983858ca)

As for the newer version of `elysia-ip`, we cannot find its declaration. I went through your changes and it seems you removed the "main" field in package.json. It is specified so in the [docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main)

**Comment by cybercoder-naj** [comment]
I am not sure whether is it a problem with my `tsconfig.json`. I followed the answers in this [thread](https://stackoverflow.com/questions/58990498/package-json-exports-field-not-working-with-typescript) but I had no luck.

Under `node_modules/elysia-ip`, I changed your package.json to include this line and it solves the issue.
```json
  "main": "./dist/index.js"
```
I don't know how `exports` work with `package.json` but it seems it is not compatible with all module types.
